rhsm.go

```diff
@@ -132,6 +133,13 @@ func registerUsernamePassword(username, password, organization, serverURL string
 		return orgs, err
 	}
 
+	var enabledContentStr string
+	if enableContent {
+		enabledContentStr = "true"
+	} else {
+		enabledContentStr = "false"
+	}
```

Comment:
I believe this could be simplified into a single `Sprintf` statement, using the `Stringer` implementation of the `bool` type.
```suggestion
	enableContentStr := fmt.Sprintf("%v", enableContent)
```